api/shipper.proto

```diff
@@ -26,6 +26,6 @@ service Producer {
  // The order in which concurrent requests complete is not guaranteed. Use sequential requests to
  // control ordering.
  rpc PublishEvents(messages.PublishRequest) returns (messages.PublishReply);
- // Returns a stream of acknowledgements from outputs.
- rpc StreamAcknowledgements(messages.StreamAcksRequest) returns (stream messages.StreamAcksReply);
+ // Returns the shipper's uuid and its position in the event stream.
+ rpc Info(messages.InfoRequest) returns (stream messages.InfoReply);
```

Comment:
I went much more specific with `PersistedIndex`, and explicitly called out that it's only needed for inputs that need a persisted position within their data source.

I also removed `accepted_index` from the `PersistedIndexReply` -- I was tempted to include it initially because it was easy and symmetric, but there's nothing an input could or should reasonably derive from it, and I don't want to tempt anyone to use it as some sort of proxy metric for the queue state.